src/main.rs

```diff
@@ -128,6 +135,10 @@ enum Command {
         /// Write output to file.
         #[arg(short, long)]
         out: Option<PathBuf>,
+        /// The output format.
+        /// Select `null` to suppress BibTeX output and disable checks on citation keys.
```

Comment:
I think you need to leave a space after the first line, or it creates an overflowed help message when you run `autobib help get`.